holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1219 @@
+        logger.exception(e)
+        raise
+
+    finally:
+        return logger.close()
+
+################################################################################
+
+def move_old_links(shelf_file, logger=None):
+    """Move a file to the /logs/ directory tree and append a time tag."""
+
+    if not os.path.exists(shelf_file): return
```

Flake8 (7.2.0)

[error] multiple statements on one line (colon)

(E701)